docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgcron.md

```diff
@@ -0,0 +1,27 @@
+---
+title: pg_cron extension
+headerTitle: pg_cron extension
+linkTitle: pg_cron
+description: Using the pg_cron extension in YugabyteDB
+menu:
+  preview:
+    identifier: extension-pgcron
+    parent: pg-extensions
+    weight: 20
+type: docs
+---
+
+The [pg_cron](https://github.com/citusdata/pg_cron) extension provides a simple cron-based job scheduler that runs inside the database. It uses the same syntax as regular cron, but it allows you to schedule YSQL commands directly from the database. You can also use '[1-59] seconds' to schedule a job based on an interval.
+
+To enable the extension:
+This feature is {{<badge/tp>}}. Use the `enable_pg_cron` flag to enable the feature. This is a preview flag so you will have to first add it to `allowed_preview_flags_csv`.
```

Comment:
```suggestion
This feature is {{<badge/tp>}}. Use the `enable_pg_cron` flag to enable the feature. As it is a preview flag, you have to first add it to `allowed_preview_flags_csv`.
```

Reply:
Can you provide the command to add it to "allowed_preview_flags_csv" ? For example, we have it on the connection manager page in the How to use section - https://docs.yugabyte.com/preview/explore/going-beyond-sql/connection-mgr-ysql/#how-to-use